charlesinto/iReporter
# #162598733 Update Red-flag status

#### What does this PR do?

- Update Red flag status

#### Description of Task to be completed?
- Implement JWT authentication
- Verify User Role
- Update Status if Admin

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/red-flags/:id/status`
- You should receive a successful response

#### Any background context you want to provide?
Request should be structure like:

`{status: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162598733](https://www.pivotaltracker.com/story/show/162598733)

#### Screenshots (if appropriate)


![image](https://user-images.githubusercontent.com/26615085/49848594-13187a00-fdd6-11e8-87d4-3be85c1c6ce5.png)





#### Questions:
N/A
